base58/src/lib.rs

```diff
@@ -73,18 +73,18 @@ pub fn decode(data: &str) -> Result<Vec<u8>, InvalidCharacterError> {
     // Build in base 256
```

Comment:
FTR there's a construction of `Vec` with useless pre-zeroing above this line. #3131